SignalServiceKit/src/Messages/OWSMessageManager.m

```diff
@@ -292,15 +294,16 @@ - (void)handleIncomingEnvelope:(OWSSignalServiceProtosEnvelope *)envelope
             OWSSyncGroupsRequestMessage *syncGroupsRequestMessage =
                 [[OWSSyncGroupsRequestMessage alloc] initWithThread:thread groupId:groupId];
             [self.messageSender sendMessage:syncGroupsRequestMessage
-                transaction:transaction
```

Comment:
why not use the same transaction?

Reply:
See the big huge comment (that we discussed above) in `OWSMessageSender`.